app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsCardView.kt

```diff
@@ -29,23 +29,26 @@ class SuggestedEditsCardView(context: Context) : DefaultFeedCardView<SuggestedEd
     var isTranslation: Boolean = false
     var sourceSummary: RbPageSummary? = null
     var targetSummary: RbPageSummary? = null
-    var addedDescription: String? = null
 
     init {
-        inflate(getContext(), R.layout.fragment_suggested_edits_add_descriptions_item, this)
+        inflate(getContext(), R.layout.view_suggested_edit_card, this)
     }
 
     override fun setCard(@NonNull card: SuggestedEditsCard) {
         super.setCard(card)
 
-        prepareViews()
-
         isTranslation = card.isTranslation
         sourceSummary = card.sourceSummary
         targetSummary = card.targetSummary
 
         setLayoutDirectionByWikiSite(WikiSite.forLanguageCode(sourceSummary!!.lang), this)
 
+        cardView.setOnClickListener {
+            if (callback != null && card != null) {
```

Comment:
the `card` is always `NonNull`, so there's no need to check the value

Reply:
Done